src/main/java/com/squareup/phrase/Phrase.java

```diff
@@ -67,6 +68,12 @@
   private char curChar;
   private int curCharIndex;
 
+  /**
+   * When enabled, span support is disabled. This can be useful for unit tests that don't want
```

Comment:
first sentence is redundant with field name.  Just keep the second sentence ("This can be useful...").